src/index.scss

```diff
@@ -30,346 +37,268 @@ $mobile-width: 750px;
-		background-color: $blue;
+			color: white;
 
-		ul li {
-			padding: 25px 15px;
-			a {
-				color: white;
-				border-bottom-style: solid;
-				border-color: transparentize(white, 1);
+			font-size: 1.25vw;
+			font-weight: 400;
 
-				&:hover {
-					border-color: transparentize(white, 1);
-				}
-				&:active {
-					border-color: transparentize(white, 0);
-				}
+			transition: font-weight 0.1s, color 0.2s, border-color 0.3s;
+			
+			&:hover {
+				font-weight: 600;
+				color: $orange;
+				border-color: transparentize($orange, 0);
 			}
-		}
+		}	
 	}
 
-	@include mobile {
-		/*no scaling for very short devices,
-		scaling matching the mlh badge scaling for avg devices,
-		high scaling for very tall devices
-		*/
-
-		margin-bottom: max(1px, calc(10vh - 40px), calc(30vh - 190px));
+	#mlh-badge {
+		position: absolute;
+		top: 0;
+		height: 13.5vw;
+		right: 2vw;
 	}
-}
 
-.section {
-	height: 100%;
-	display: flex;
-	flex-direction: column;
-	
-	@include desktop {
-		min-height: 750px;
-	}
-	@include tablet {
-		height: calc(100% - 74.5px);
-	}
 	@include mobile {
-		height: 0%;
-	}
-}
-
-.content {
-	width: 100%;
-	
-	max-width: 1050px;
-	text-align: center;
-	margin: auto;
-
-	@include tablet {
-		max-width: 100%;
-		flex-direction: column;
-		margin: auto;
-	}
-}
-
-.right-half {
-	flex: 50%;
-}
-.left-half {
-	@extend .right-half;
-	margin-bottom: 2.5vh;
-
-	@include tablet {
-		margin-bottom: 15vh;
+		background-color: transparentize(white, 0.85);
+		height: 20vw;
+		
+		#mlh-badge {
+			top: 0;
+			height: 28vw;
+			left: 5vw;
+		}
 	}
 }
 
-.hero-img {
-	width: 100%;
+section {
+	position: relative;
+	display: block;
 
-	@include mobile {
-		/* slightly smaller hero img for shorter screens */
-		min-width: 80%;
-		width: min(50vh, 90%);
-	}
-}
+	min-height: 55vw; // min. ratio of ~1.66 slightly less than 16:9 (1.77) to account for browser window, etc.
+	max-height: 75vw; // max. ratio of ~1.33 to handle horizontal tablet sizing (ipad)
 
-.hero-img.mobile-hide { //temporary, push up the desktop hero image a bit
-	margin-bottom: 75px;
-}
+	height: 100vh;
+	width: 100%; // cut off background image
 
-/*
-note: development on a mac may not have all font-weights,
-lines of text may be wider than they appear
-*/
-#hackathon-name {
-	height: 75px;
-	font-size: 70px;
-	font-weight: 700;
-	color: $blue;
+	background-color: $dark-blue;
 
 	@include mobile {
-		height: 14.1vw;
-		font-size: 13vw;
+		max-height: 100vh;
+		height: 100vh;
 	}
 }
-#hackathon-description {
-	height: 43px;
-	font-size: 34px;
-	font-weight: 700;
-	color: $blue;
 
-	@include mobile {
-		height: 8.2vw;
-		font-size: 6.3vw;
-	}
-}
-#hackathon-info {
-	height: 25px;
-	font-size: 17px;
-	font-weight: 500;
-	color: $blue;
+#general {
+	background-color: $light-blue;
+	
+	background-image: url($assets+'shelf.svg'), url($assets+'desk.svg');
+	background-repeat: no-repeat, no-repeat;
+	background-size: 100vw, 85vw;
+	background-position: 0 130%, 24.2vw 75%;
```

Comment:
I'm a bit conflicted on the mixed units here: what's preventing the use of just `vw`?